### PR TITLE
Feature/rename tracker to type

### PR DESCRIPTION
**Pull request by apfelfabrik**
This PR renames the model Tracker into Type to prepare the succeeding merge of PlanningElementType into Type.

### CODE REVIEW

File: config/locales/de.yml [hunk 1]
```diff
@@ -469,7 +469,7 @@ de:
 
   error_can_not_archive_project: "Dieses Projekt kann nicht archiviert werden."
   error_can_not_delete_custom_field: "Kann das benutzerdefinierte Feld nicht löschen."
-  error_can_not_delete_tracker: "Dieser Tracker enthält Tickets und kann nicht gelöscht werden."
+  error_can_not_delete_type: "Dieser Type enthält Tickets und kann nicht gelöscht werden."
```
Comment (hschink) [inline config/locales/de.yml]: This should be `Typ` and not `Type`

File: config/locales/de.yml [hunk 2]
```diff
@@ -478,14 +478,14 @@ de:
   error_issue_not_found_in_project: "Das Ticket wurde nicht gefunden oder gehört nicht zu diesem Projekt."
   error_must_be_project_member: "muss Mitglied des Projekts sein"
   error_no_default_issue_status: "Es ist kein Status als Standard definiert. Bitte überprüfen Sie Ihre Konfiguration (unter \"Administration -> Ticket-Status\")."
-  error_no_tracker_in_project: "Diesem Projekt ist kein Tracker zugeordnet. Bitte überprüfen Sie die Projekteinstellungen."
+  error_no_type_in_project: "Diesem Projekt ist kein Type zugeordnet. Bitte überprüfen Sie die Projekteinstellungen."
```
Comment (hschink) [inline config/locales/de.yml]: This should be `Typ` and not `Type`

File: config/locales/de.yml [hunk 3]
```diff
@@ -478,14 +478,14 @@ de:
...
   error_scm_annotate: "Der Eintrag existiert nicht oder kann nicht annotiert werden."
   error_scm_command_failed: "Beim Zugriff auf das Projektarchiv ist ein Fehler aufgetreten: %{value}"
   error_scm_not_found: "Eintrag und/oder Revision existiert nicht im Projektarchiv."
   error_unable_delete_issue_status: "Der Ticket-Status konnte nicht gelöscht werden."
   error_unable_to_connect: "Fehler beim Verbinden (%{value})"
-  error_workflow_copy_source: "Bitte wählen Sie einen Quell-Tracker und eine Quell-Rolle."
-  error_workflow_copy_target: "Bitte wählen Sie die Ziel-Tracker und -Rollen."
+  error_workflow_copy_source: "Bitte wählen Sie einen Quell-Type und eine Quell-Rolle."
+  error_workflow_copy_target: "Bitte wählen Sie die Ziel-Type und -Rollen."
```
Comment (hschink) [inline config/locales/de.yml]: This should be `Typ` and not `Type`

File: config/locales/de.yml [hunk 4]
```diff
@@ -478,14 +478,14 @@ de:
...
-  error_no_tracker_in_project: "Diesem Projekt ist kein Tracker zugeordnet. Bitte überprüfen Sie die Projekteinstellungen."
+  error_no_type_in_project: "Diesem Projekt ist kein Type zugeordnet. Bitte überprüfen Sie die Projekteinstellungen."
   error_scm_annotate: "Der Eintrag existiert nicht oder kann nicht annotiert werden."
   error_scm_command_failed: "Beim Zugriff auf das Projektarchiv ist ein Fehler aufgetreten: %{value}"
   error_scm_not_found: "Eintrag und/oder Revision existiert nicht im Projektarchiv."
   error_unable_delete_issue_status: "Der Ticket-Status konnte nicht gelöscht werden."
   error_unable_to_connect: "Fehler beim Verbinden (%{value})"
-  error_workflow_copy_source: "Bitte wählen Sie einen Quell-Tracker und eine Quell-Rolle."
-  error_workflow_copy_target: "Bitte wählen Sie die Ziel-Tracker und -Rollen."
+  error_workflow_copy_source: "Bitte wählen Sie einen Quell-Type und eine Quell-Rolle."
```
Comment (hschink) [inline config/locales/de.yml]: This should be `Typ` and not `Type`

File: config/locales/de.yml [hunk 5]
```diff
@@ -655,7 +655,7 @@ de:
   label_display: "Anzeige"
   label_display_per_page: "Pro Seite: %{value}"
   label_display_sums: "Summen anzeigen"
-  label_display_used_statuses_only: "Zeige nur Status an, die von diesem Tracker verwendet werden"
+  label_display_used_statuses_only: "Zeige nur Status an, die von diesem Type verwendet werden"
```
Comment (hschink) [inline config/locales/de.yml]: This should be `Typ` and not `Type`

File: config/locales/de.yml [hunk 6]
```diff
@@ -897,8 +897,8 @@ de:
   label_top_menu: "Hauptmenu"
   label_topic_plural: "Themen"
   label_total: "Gesamtzahl"
-  label_tracker_new: "Neuer Tracker"
-  label_tracker_plural: "Tracker"
+  label_type_new: "Neuer Type"
```
Comment (hschink) [inline config/locales/de.yml]: This should be `Typ` and not `Type`

File: config/locales/de.yml [hunk 7]
```diff
@@ -897,8 +897,8 @@ de:
...
+  label_type_plural: "Type"
```
Comment (hschink) [inline config/locales/de.yml]: This should be `Typen` and not `Type`

File: config/locales/de.yml [hunk 8]
```diff
@@ -1274,7 +1274,7 @@ de:
   text_line_separated: "Mehrere Werte sind erlaubt (eine Zeile pro Wert)."
   text_load_default_configuration: "Standard-Konfiguration laden"
   text_min_max_length_info: "0 heißt keine Beschränkung"
-  text_no_configuration_data: "Rollen, Tracker, Ticket-Status und Workflows wurden noch nicht konfiguriert.\nEs ist sehr zu empfehlen, die Standard-Konfiguration zu laden. Sobald sie geladen ist, können Sie sie abändern."
+  text_no_configuration_data: "Rollen, Type, Ticket-Status und Workflows wurden noch nicht konfiguriert.\nEs ist sehr zu empfehlen, die Standard-Konfiguration zu laden. Sobald sie geladen ist, können Sie sie abändern."
```
Comment (hschink) [inline config/locales/de.yml]: This should be `Typ` and not `Type`

File: config/locales/de.yml [hunk 9]
```diff
@@ -1292,7 +1292,7 @@ de:
   text_tip_issue_begin_day: "Aufgabe, die an diesem Tag beginnt"
   text_tip_issue_begin_end_day: "Aufgabe, die an diesem Tag beginnt und endet"
   text_tip_issue_end_day: "Aufgabe, die an diesem Tag endet"
-  text_tracker_no_workflow: "Kein Workflow für diesen Tracker definiert."
+  text_type_no_workflow: "Kein Workflow für diesen Type definiert."
```
Comment (hschink) [inline config/locales/de.yml]: This should be `Typ` and not `Type`

**Comment by apfelfabrik** [comment]
I have fixed the German translations and merged w/ feature/rails3 once more.

**Comment by apfelfabrik** [comment]
Fixed another cuke that broke after df3235c.

**Pull request merged**